### PR TITLE
Remove tigrannajaryan as a maintainer

### PR DESCRIPTION
**Pull request by tigrannajaryan**
I am no longer able to allocate time to be a maintainer of the operator
due to focusing on other parts of OpenTelemetry.

I am moving myself to the Emeritus section according to the rules introduced in
https://github.com/open-telemetry/community/pull/961

I have also listed approvers and maintainers directly in the README, like we
do in the Collector repos. I will also remove Collector github teams' permissions
from this repo since we have github teams for operator to avoid confusion as to
who has what permissions.

### CODE REVIEW

**Comment by tigrannajaryan** [comment]
> I would consider to have all approver/maintainers from collector as emeritus, and let the current active maintainers decide on who to keep as active approver/maintainers.

I don't want to mess with others' permissions :-) Let's do in separate PR?

**Review by bogdandrutu — APPROVED** [review]

**Comment by bogdandrutu** [comment]
@tigrannajaryan once this is merged I will create a new PR to propose more moves to emeritus :)

**Review by codeboten — APPROVED** [review]

**Comment by bogdandrutu** [comment]
@open-telemetry/operator-maintainers merging this, since no changes to the list of active roles, except @tigrannajaryan move to emeritus which I think is fine to do with his approval.

**Comment by tigrannajaryan** [comment]
I removed collector-approvers and collector-maintainers from the list of teams that are permitted in this repo and instead added everyone to operator-approvers and operator-maintainers teams. This should result in no changes in permissions (except I removed myself from operator).

**Pull request merged**
